tests/contrib/openstack/test_os_contexts.py

```diff
@@ -1211,9 +1216,8 @@ def test_identity_service_context_with_missing_relation(self, *args):
     @patch.object(context, 'filter_installed_packages')
     @patch.object(context, 'os_release')
     def test_keystone_authtoken_www_authenticate_uri_stein_apiv3(self, mock_os_release, mock_filter_installed_packages):
-        relation_data = deepcopy(IDENTITY_SERVICE_RELATION_UNSET)
-        relation_data['api_version'] = '3'
-        relation = FakeRelation(relation_data=relation_data)
+        relation = FakeRelation(
+            relation_data=IDENTITY_SERVICE_RELATION_VERSIONED)
```

Comment:
Why did you remove the deepcopy?

Reply:
Because I'm not changing the data

Reply:
But it was a good defensive programming strategy.  As it's a constant, it's a good idea to deepcopy it for a test.  that's why I was curious.